src/core_codemods/sonar/results.py

```diff
@@ -36,7 +37,21 @@ def from_result(cls, result) -> Self:
             ]
             for flow in result.get("flows", [])
         ]
-        return cls(rule_id=rule_id, locations=locations, codeflows=all_flows)
+
+        return cls(
+            finding_id=rule_id,
+            rule_id=rule_id,
+            locations=locations,
+            codeflows=all_flows,
+            finding=Finding(
+                id=rule_id,
+                rule=Rule(
+                    id=rule_id,
+                    name=rule_id,
+                    url=None,
```

Comment:
I think we _could_ do the URL here but I'm fine to leave it as a separate issue. It would be something like
```python
url="https:/r/ules.sonarsource.com/python/RSPEC-<rule-id-here>/"
```

Reply:
unfortunately I don't think we've yet figured out why sonar urls are inconsistent. grep `rules.sonarsource.com/` in our code and you see stuff like
https://rules.sonarsource.com/python/type/Vulnerability/RSPEC-5144/
https://rules.sonarsource.com/python/type/Bug/RSPEC-6725/

so either there is a generalized url for all rules and we just copied the wrong thing or there is no generalized rule url.